db/dumps/kohdataan.pgsql

```diff
@@ -125,7 +125,8 @@ CREATE TABLE public."Users" (
     location character varying(255),
     description character varying(255),
     "profileReady" boolean NOT NULL,
-    "tutorialWatched" boolean NOT NULL
```

Comment:
I think this should either remain as it is (old) :D or then just create new dump using pg_dump command line tool

Reply:
So I didn't touch this. Not sure how to add the old dump back in..